include/swift/AST/RequestCache.h

```diff
@@ -29,10 +29,6 @@ namespace evaluator {
 
 namespace detail {
 
-// Remove this when the compiler bumps to C++17.
-template <typename...> using void_t = void;
-template<typename T, typename = void_t<>>
-
```

Comment:
Personally I have a preference for headers including the correct headers for things they use, so this one should probably now `#include <type_traits>`; I note, though that it's already missing `<tuple>` and probably other things, so it's not unlikely it currently builds OK.